README.md

```diff
@@ -7,9 +7,9 @@
 * Pulling the results into the template
 
 ## Setting expectations: Searching vs. filtering
-Before we begin, it's important that we set a baseline for what we're looking to accomplish in this tutorial, because search, as you may know, is a real can of worms.
+Before we begin, it's important that we set a baseline for what we're looking to accomplish in this tutorial. Because search, as you may know, is a real can of worms.
```

Comment:
Personally I think it was better the way it was, but perhaps it might be better again as "...in this tutorial: search, as you know..."